### PR TITLE
Updating old permissions

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- https://linear.app/basis-theory/issue/ENG-1885/update-permissions-in-guides-and-sample-app

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/guides/collect_payments/index.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ To set up a Reactor, head over to our Portal and set up a new Stripe Reactor. If
 ## Using your new Reactor 
 <span class="base-alert warning">
   <span>
-    To use your Stripe Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>card:read</code>, <code>card:create</code>, and <code>reactor:read</code>.
+    To use your Stripe Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>token:pci:read:high</code>, <code>token:pci:create</code> and <code>token:pci:use:reactor</code>.
```
Comment (koalaty-code) [inline collections/guides/collect_payments/index.md:90]:
```suggestion
    To use your Stripe Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>token:pci:read:low</code>, <code>token:pci:create</code> and <code>token:pci:use:reactor</code>.
```

`token:pci:read:high` isn't an available permission without PCI attestation.

File: collections/guides/setup_first_reactor/setup_first_reactor.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ The following example is for an [Atomic Card](https://docs.basistheory.com/api-r
 
 <span class="base-alert warning">
   <span>
-    To run a Reactor, an application needs `<source_token_type>:read` and `reactor:read` permissions. For this example you will need `card:read` and `reactor:read`.
+    To run a Reactor, an application needs `token:<classification>:read:<impact_level>` and `token:<classification>:use:reactor` permissions. For this example you will need `token:pci:read:<classification>` and `token:pci:use:reactor`.
```
Comment (koalaty-code) [inline collections/guides/setup_first_reactor/setup_first_reactor.md:59]:
```suggestion
    To run a Reactor, an application needs `token:<classification>:use:reactor` permission. For this example the application will need `token:pci:use:reactor` permission.
```

File: collections/guides/tokenize_customer_data_in_browser/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Basis Theory is built to handle any payload of data you can serialize. The follo
 
 <span class="base-alert warning">
   <span>
-    You'll need a <code>Client-side</code> [Application](https://docs.basistheory.com/api-reference/#applications) created with the <code>token:create</code> permission for this step.
+    You'll need a <code>Client-side</code> [Application](https://docs.basistheory.com/api-reference/#applications) created with the <code>token:pii:create</code> permission for this step.
```
Comment (koalaty-code) [inline collections/guides/tokenize_customer_data_in_browser/index.md:28]: We also need to update the `submit()` handler example to override default Privacy Settings for `pii` classification:
```
  const token = await bt.tokens.create({
    type: 'token',
    data: {
      name,
      ssn,
      phone
    },
    privacy: {
      classification: 'pii'
    });
```

**Review by koalaty-code — CHANGES_REQUESTED** [review]

**Review by dhudec — DISMISSED** [review]

**Review by koalaty-code — APPROVED** [review]

**Review by dhudec — APPROVED** [review]

**Pull request merged**